zookeeper-server/src/main/java/org/apache/zookeeper/server/ZKDatabase.java

```diff
@@ -151,7 +158,24 @@ public ZKDatabase(FileTxnSnapLog snapLog) {
                 DEFAULT_COMMIT_LOG_COUNT);
             commitLogCount = DEFAULT_COMMIT_LOG_COUNT;
         }
-        LOG.info("{}={}", COMMIT_LOG_COUNT, commitLogCount);
+        try {
+            commitLogSize = Double.parseDouble(
```

Comment:
Will `Double.parseDouble()` accept floating point numbers from the config?
Shouldn't we prevent that?